src/Models/ProjectLabels.coffee

```diff
@@ -0,0 +1,31 @@
+BaseModel = require '../BaseModel'
+Utils = require '../Utils'
+
+class ProjectLabels extends BaseModel
+  list: (projectId, params = {}, fn = null) =>
```

Comment:
I think following the conventions we have set elsewhere, we'd call this `all()`.